src/tree/containers/ContainersTreeItem.ts

```diff
@@ -67,8 +67,26 @@ export class ContainersTreeItem extends LocalRootTreeItemBase<ILocalContainerInf
                 return item.state;
             case 'Status':
                 return item.status;
+            case 'Compose Project Name':
+                return item.compose;
             default:
                 return getImagePropertyValue(item, property);
         }
     }
+
+    public compareChildrenImpl(ti1: ContainerTreeItem, ti2: ContainerTreeItem): number {
+        if (this.failedToConnect) {
+            return 0; // children are already sorted
+        }
+        if (this.groupBySetting === 'Compose Project Name'
```

Comment:
I assume this is to ensure that non-Compose group is at the bottom. Consider adding a comment clarifying this intent.